apps/server/src/services/__tests__/timerUtils.test.ts

```diff
@@ -741,7 +740,7 @@ describe('getRuntimeOffset()', () => {
       },
     } as RuntimeState;
 
-    const offset = getRuntimeOffset(state);
+    const offset = getRuntimeOffset(state).absoluteOffset;
```

Comment:
nitpick: i think this whole test suite would become clearer if we used destructuring instead. Since we are introducing variants of offset, the word `offset` is no longer specific enough to describe a dataset